### ISSUE TITLE
Hope can customize the windows position

### ISSUE THREAD

**Issue by cccpism** [adirh3/Fluent-Search#927]
In the latest version, the evoke window can no longer appear in a customized position as in previous versions, but is only pinned to the middle of the screen, I've tried to adjust the window position in the settings, but it doesn't works.

**Expected behavior**
Wish the wakeup window would appear in a customized location

**Desktop (please complete the following information):**
 - Window 11 23H2 22635.2700
 - Fluent Search Version: 1.0.0.24
 - Installation source: Installer

**Comment by adirh3** [comment]
This issue has been potentially fixed in version 1.0.0.25                                          . Changelog - https://install.appcenter.ms/users/adirh3-gmail.com/apps/fluent-search-alpha/distribution_groups/exe

**Comment by cccpism** [comment]
> This issue has been potentially fixed in version 1.0.0.25 . Changelog - https://install.appcenter.ms/users/adirh3-gmail.com/apps/fluent-search-alpha/distribution_groups/exe

Sorry，I‘ve update（1.0.0.25），but this issue hasn't been fixed

**Comment by adirh3** [comment]
@cccpism just double checking, do you have `Window startup position` set to `Last position` in Settings -> System -> Startup?

**Comment by cccpism** [comment]
yeah , I'm sure . And to ensure this button works , I changed different settings but found the window always appeared in default position .
________________________________

**Comment by cccpism** [comment]
Yes, and I changed different settings, such as following the mouse, etc., and found that the window still only appears in the center of the screen

**Comment by adirh3** [comment]
> Yes, and I changed different settings, such as following the mouse, etc., and found that the window still only appears in the center of the screen

Any of Fluent Search's windows? Or a specific one? Do the Setting window remember the last position?

**Comment by cccpism** [comment]
![屏幕截图 2023-11-19 232416](https://github.com/adirh3/Fluent-Search/assets/93852710/cfc9357c-0d25-49d7-8649-c5afeda6bf6a)
I guess it may be clearer
